src/screencapturekit.cs

```diff
@@ -212,6 +268,24 @@ interface SCContentFilter {
 		[Internal]
 		[Export ("initWithDisplay:excludingApplications:exceptingWindows:")]
 		NativeHandle InitWithDisplayExcludingApplications (SCDisplay display, SCRunningApplication [] excludingApplications, SCWindow [] exceptingWindows);
+
+		// per docs, the following selectors are available for 12.3+
+		// but return types are SCStreamType and SCShareableContentStyle are 14.0+
```

Comment:
thoughts? all tests passed locally via this approach and i think it makes sense since these api were added as part of this Xcode release, maybe a bug on apple's end?

Reply:
Usually the new apis will be the latest version and in this case, I think this is the safer and more correct option

Reply:
Yup you made the right call, sometimes apple makes mistakes with availability attributes there are very few exceptions to this rule but I think it makes sense as is right now.

Reply:
It is safer as @tj-devel709 mentioned to go high. Going low and then having runtime issues in the customers side is way worse than them getting a warning they can ignore, get an issue and fix it. In case of doubt, always latests.

Reply:
good to know, thanks all